pom.xml

```
@@ -288,7 +288,7 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
+      <artifactId>slf4j-reload4j</artifactId>
```

Would this work instead? not sure if it's being used as a bridge or as an implementation
```suggestion
      <artifactId>slf4j-jdk14</artifactId>
```

---

According to the SLF4J manual, the preferred drop-in replacement is reload4j.

---

but do we want this to stay on log4j 1?

---

Quickly checked locally, jdk14 passes `verify`.